cypress/plugins/index.js

```diff
@@ -95,49 +98,34 @@ module.exports = (on, config) => {
             reject);
       });
     },
+
     /**
-     * Recursively deletes all data under the test/currentTestRoot tree, and
-     * creates necessary data for tests to consume, pulling that data from prod
-     * Firebase. For use before a test case runs. We add a dummy field inside
-     * test/currentTestRoot so that Firestore will deign to list this document
-     * later
-     * (https://stackoverflow.com/questions/47043651/this-document-does-not-exist-and-will-not-appear-in-queries-or-snapshots-but-id)
-     * @param {string} currentTestRoot document name directly underneath test/
-     * @return {Promise<null>} Promise that resolves when deletion and data
-     * creation is complete
+     * Writes disasters data (retrieved using {@link storeTestFirestoreData}
```

Comment:
Can you add a comment about when this is called (the one in storeTestFirestoreData already helpful)

Reply:
Done.